### PR TITLE
Update PSR-7-http-message-meta.md

### PR DESCRIPTION
**Pull request by WinterSilence**
Normalize examples

### CODE REVIEW

File: accepted/PSR-7-http-message-meta.md [hunk 1]
```diff
@@ -393,8 +393,8 @@ headers:
 
 ~~~php
 // where Stream is a concrete StreamInterface:
-$stream = new Stream($filename);
-$finfo  = new finfo(FILEINFO_MIME);
+$stream   = new Stream($filename);
```
Comment (drupol) [inline accepted/PSR-7-http-message-meta.md]: Nope.

File: accepted/PSR-7-http-message-meta.md [hunk 2]
```diff
@@ -393,8 +393,8 @@ headers:
...
+$finfo    = new finfo(FILEINFO_MIME);
```
Comment (drupol) [inline accepted/PSR-7-http-message-meta.md]: Nope

File: accepted/PSR-7-http-message-meta.md [hunk 3]
```diff
@@ -252,26 +252,27 @@ request or reset request state for each message the client sends:
...
-$baseRequest = new Request($uri, null, [
-    'Authorization' => 'Bearer ' . $token,
-    'Accept'        => 'application/json',
-]);;
+$baseRequest = new Request(
+    $uri, 
+    null, 
+    [
+        'Authorization' => 'Bearer ' . $token,
+        'Accept'        => 'application/json',
```
Comment (drupol) [inline accepted/PSR-7-http-message-meta.md]: I don't think that using that rule here is relevant.

Comment (WinterSilence) [reply accepted/PSR-7-http-message-meta.md]: replace to ...?

File: accepted/PSR-7-http-message-meta.md (hunk 2 above)
Comment (WinterSilence) [reply accepted/PSR-7-http-message-meta.md]: already fixed

File: accepted/PSR-7-http-message-meta.md (hunk 3 above)
Comment (drupol) [reply accepted/PSR-7-http-message-meta.md]: Remove empty extra spaces

Comment (WinterSilence) [reply accepted/PSR-7-http-message-meta.md]: fixed

File: accepted/PSR-7-http-message-meta.md [hunk 4]
```diff
@@ -252,26 +252,24 @@ request or reset request state for each message the client sends:
...
 ~~~php
 $uri = new Uri('http://api.example.com');
-$baseRequest = new Request($uri, null, [
-    'Authorization' => 'Bearer ' . $token,
-    'Accept'        => 'application/json',
-]);;
+$baseRequest = new Request(
+    $uri, 
+    null, 
+    ['Authorization' => 'Bearer ' . $token, 'Accept' => 'application/json']
```
Comment (drupol) [inline accepted/PSR-7-http-message-meta.md]:
```suggestion
    [
        'Authorization' => 'Bearer ' . $token,
        'Accept' => 'application/json',
    ]
```

File: accepted/PSR-7-http-message-meta.md [hunk 5]
```diff
@@ -252,26 +252,24 @@ request or reset request state for each message the client sends:
...
 $request = $baseRequest->withUri($uri->withPath('/user'))->withMethod('GET');
 $response = $client->send($request);
 
 // get user id from $response
 
-$body = new StringStream(json_encode(['tasks' => [
-    'Code',
-    'Coffee',
-]]));;
+$body = new StringStream(json_encode(['tasks' => ['Code', 'Coffee']]));
```
Comment (drupol) [inline accepted/PSR-7-http-message-meta.md]:
```suggestion
$body = new StringStream(
    json_encode(
        [
            'tasks' => [
                'Code', 
                'Coffee',
            ]
        ]
    )
);
```

Comment (WinterSilence) [reply accepted/PSR-7-http-message-meta.md]: > Lines SHOULD NOT be longer than 80 characters; lines longer than that SHOULD be split into multiple subsequent lines of no more than 80 characters each.

whats wrong in current example?

**Comment by WinterSilence** [comment]
@drupol About whitespaces: 
We can talk only about my changes. I'm save whitespaces "as is" and you can't say "it's wrong/fix it" because it's not "part" of current PR.

File: accepted/PSR-7-http-message-meta.md (hunk 5 above)
Comment (drupol) [reply accepted/PSR-7-http-message-meta.md]: It's just to be consisted with the rest of the PR. To be decided later maybe.

Comment (WinterSilence) [reply accepted/PSR-7-http-message-meta.md]: don't understand :(

File: accepted/PSR-7-http-message-meta.md [hunk 6]
```diff
@@ -252,30 +252,31 @@ request or reset request state for each message the client sends:
...
-    'Accept'        => 'application/json',
-]);;
+$baseRequest = new Request(
+    $uri, 
+    null, 
+    [
+        'Authorization' => 'Bearer ' . $token,
+        'Accept' => 'application/json',
+    ]
+);
```
Comment (Jean85) [inline accepted/PSR-7-http-message-meta.md]: This seems an unnecessary white-space edit for me, it was fine.

File: accepted/PSR-7-http-message-meta.md (hunk 5 above)
Comment (Jean85) [reply accepted/PSR-7-http-message-meta.md]: I'm against this suggested change, it puts too much focus on the unimportant part of the example.

**Comment by WinterSilence** [comment]
@drupol @Jean85 whitespaces: resolve your dispute p2p, I did not plan these changes and for me they are not important.

**Comment by drupol** [comment]
It's ok, I don't mind one solution or the other, it's just my personal taste.

**Comment by WinterSilence** [comment]
@drupol @Jean85 then I will return the spaces to original, ok?

@drupol It didn't sound like a recommendation/wish: "need do so" it's not "i would do so". I think that you did not want to impose your opinion on anyone, but there is a misunderstanding in network correspondence.

**Comment by drupol** [comment]
Sure, sorry about the misunderstanding!

**Comment by vdelau** [comment]
@weierophinney, as editor of PSR-7 can you accept or reject these changes?

**Pull request closed without merging**